### ISSUE TITLE
Drilling Advancement

### ISSUE THREAD

**Issue by VrMahmoudabadi** [DIGGSml/diggs-schema#38]
There are 2 issues with the Advancement element in the schema:

1- There is no place to map the "Readings set reference [AGS Standard field]" in the element, unless it goes to gml:description that I don't believe it is a right place to use. It seems additional field needs to be created for this data. 

2- What is the difference between "AdvancementRate" and "PenetrationRate"? Both have very similar description and can be used for one purpose which is the penetration rate.

**Comment by VrMahmoudabadi** [comment]
Regarding the "energyDensity":

1- what is the physical meaning of this property in the element?
2- How does this apply to MWD?
3- The uom is not defined for this property in version 2.5.a.

**Comment by dponti** [comment]
In 2.6-dev, drillAdvancement property in Borehole is now of ArrayProperty type that has a setName attribute, meaning that you can list a series of Advancement objects under one drillAdvancement property tag and assign a name to the set of advancement readings. This covers the use case were intervals of the hole may be redrilled (the use case that AGS uses reading set reference).

**Comment by dponti** [comment]
PenetrationRate is deprecated in 2.6-dev. Use AdvancementRate instead. This duplication was not caught in 2.5.a

**Comment by dponti** [comment]
Energy density is (I believe) an inappropriate term - it should be specificEnergy or better mechanicalSpecificEnergy. This explains why the UOM is incorrect. in 2.6-dev, mechanical enSpecificEnergy property is added and energyDensity is deprecated.

With respect to MWD, the Advancement object is used to summarize drill advancement information over an interval of teh hole. If real-time MWD information is recorded, this is best handled through a different object that can allow compact encoding of MWS data as a time series.

**Comment by sdeaton75** [comment]
I agree that for MWD data, we really need a more compact format - like the CPT data to properly encode this from automated data acquisition otherwise the file will be massive.  I would suggest we add this in 2.6-dev. I will compile a list of properties and run it by one of the MWD drill rig manufacturers and get their feedback.
